public/auth.js

```diff
@@ -0,0 +1,20 @@
+class AuthController {
+    // Authentication state listener
+    static get_user() {
```

Comment:
```suggestion
    static getUser() {
```
(to follow JS naming convention)